examples/videocomposer/run_train.sh

```diff
@@ -4,12 +4,16 @@ export ASCEND_GLOBAL_LOG_LEVEL=1  # Global log message level for Ascend. Setting
 export ASCEND_SLOG_PRINT_TO_STDOUT=0 # 1: detail, 0: simple
 export DEVICE_ID=0  # The device id to runing training on
 
+# uncomment this following line for caching and loading the compiled graph
+#export MS_COMPILER_CACHE_ENABLE=1
+#export MS_COMPILER_CACHE_PATH=/cache
```

Comment:
We have these boilerplates in almost every script. I think it will be nice if we create a `set_env.sh` script where all important environmental variables are set, and then can call execution of this script in each train or run script. This is just a comment for the future...